Revisions for Pathogenesis from @alavendelm (#790)
greenelab/covid19-review: content/07.pathogenesis.md

However, despite the high level of sequence homology, certain protein structures might be immunologically distinct, which would be likely to prohibit effective cross-neutralization across different SARS species [@doi:10.1101/2020.03.21.990770].
Consequently, proteomic analyses of SARS-CoV-1 might also provide some essential information regarding the new pathogen [@doi:10.1073/pnas.0407992101; @doi:10.1016/j.bbrc.2004.02.074].

> Considering the paucity of omics-level big data sets for SARS-CoV-2 currently available, existing data hubs that contain information for other coronaviruses such as UniProt, NCBI Genome Database, The Immune Epitope Database and Analysis Resource (IEDB), and The Virus Pathogen Resource (ViPR) will serve as useful resources comparative bioinformatics research of SARS-CoV-2.

Author: ```suggestion
Considering the paucity of omics-level big data sets for SARS-CoV-2 currently available, existing data hubs that contain information for other coronaviruses such as UniProt, NCBI Genome Database, The Immune Epitope Database and Analysis Resource (IEDB), and The Virus Pathogen Resource (ViPR) will serve as useful resources for comparative bioinformatics research of SARS-CoV-2.
```

This finding could explain the exacerbation of the immune response that shapes the pathology and the high cytokine levels characteristic of COVID-19, possibly due to the chemotaxis of neutrophils mediated by IL-8 and IL-6.
Finally, it was suggested [@doi:10.1101/2020.04.09.033522] that the E protein of both SARS-CoV-1 and SARS-CoV-2 has a conserved Bcl-2 Homology 3 (BH3)-like motif, which could inhibit anti-apoptosis proteins, e.g., BCL2, and trigger the apoptosis of T cells.
Several compounds are known to disrupt the host-pathogen protein interactome, largely through the inhibition of host proteins.

> Therefore, preliminary research in the proteomics of SARS-CoV-2 infection suggests that drugs modulating the protein-level interactions between virus and host might be worth investigating. 

Author: @alavendelm says: 

> not a strong ending. Is this not obvious? Can it be replaced with something specific?

Author: ```suggestion
Therefore, preliminary research in the proteomics of SARS-CoV-2 infection suggests that drugs modulating the protein-level interactions between virus and host might be worth investigating.
```

### Dynamics of Transmission

Disease spread dynamics can be estimated using R~0~, the basic reproduction number, and R~t~, the effective reproduction number.
Accurate estimates of both are crucial to understanding the dynamics of infection and to predicting the effects of different interventions.
R~0~ is the average number of new (secondary) infections caused by one infected person, assuming a wholly susceptible population [@doi:10.1111/j.1467-9574.1996.tb01482.x] and is one of the most important epidemiological parameters [171].
A simple mechanistic model used to describe infectious disease dynamics is a susceptible-infected-recovered (SIR) compartmental model [@doi:10.1098/rspa.1927.0118; @doi:10.1038/280361a0].

> In this model individuals move through three states: susceptible, infected, and recovered; two parameters, $\gamma$ and $\beta$, specify the rate at which the infectious recover, and the infection transmission rate, respectively.

Author: ```suggestion
In this model, individuals move through three states: susceptible, infected, and recovered; two parameters, $\gamma$ and $\beta$, specify the rate at which the infectious recover, and the infection transmission rate, respectively.
```

### Dynamics of Transmission

Disease spread dynamics can be estimated using R~0~, the basic reproduction number, and R~t~, the effective reproduction number.
Accurate estimates of both are crucial to understanding the dynamics of infection and to predicting the effects of different interventions.

> R~0~ is the average number of new (secondary) infections caused by one infected person, assuming a wholly susceptible population [@doi:10.1111/j.1467-9574.1996.tb01482.x] and is one of the most important epidemiological parameters [171].

Reviewer: ```suggestion
R~0~ is the average number of new (secondary) infections caused by one infected person, assuming a wholly susceptible population [@doi:10.1111/j.1467-9574.1996.tb01482.x] and is one of the most important epidemiological parameters [@isbn:9780691116174].
```
Is this the correct reference to add back?

Author: 9780691116174 is correct! Thank you for catching this!

A pathogen can invade a susceptible population only if R~0~ > 1 [@isbn:9780691116174; @isbn:9780199209996].
The spread of an infectious disease at a particular time t can be quantified by R~t~, the effective reproduction number, which assumes that part of the population has already recovered (and thus gained immunity to reinfection) or that mitigating interventions were put into place.
The spread of an infectious disease at a particular time t can be quantified by R~t~, the effective reproduction number, which assumes that part of the population has already recovered (and thus gained immunity to reinfection) or that mitigating interventions have been put into place.
For example, if only a fraction S~t~ of the population is still susceptible, R~t~ = S~t~ x R~0~.
When R~t~ is greater than 1, an epidemic grows (i.e., the proportion of the population that is infectious increases); when R~t~ is less than 1, the proportion of the population that is infectious decreases.
R~0~ and R~t~ can be estimated directly from epidemiological data or inferred using mathematical modeling.

> Modeling approaches are typically based upon a classic epidemiological model structure: the SIR model and its extensions [@https://doi.org/10.1098/rspa.1927.0118; @doi:10.1038/280361a0].

Reviewer: Were these two references intentionally removed?

Author: I double-checked and those are the changes I received!

Many current efforts to model R~t~ have also led to tools that assist the visualization of estimates in real time or over recent intervals [@url:https://epiforecasts.io/covid; @url:https://rt.live].
These are valuable resources, yet it is also important to note that the estimates arise from models containing many assumptions and are dependent on the quality of the data they use, which varies widely by region.

### Molecular Signatures and Transmission

Genetic variation in SARS-CoV-2 has been used to elucidate patterns over time and space.
Mutations observed in individual SARS-CoV-2 genome sequences can be used to trace transmission patterns and have provided insights during outbreak investigations [@doi:10.1186/s12967-020-02344-6; @doi:10.1093/nsr/nwaa036; @doi:10.1016/j.meegid.2020.104351].
Similar mutations observed in several patients may indicate that the patients belong to the same transmission group.
The tracking of SARS-CoV-2 mutations is recognized as an essential tool for controlling future outbreaks and tracing the path of the spread of SARS-CoV-2.

> Efforts vary widely by country: the UK has coordinated has coordinated a national database of viral genomes [@doi:10.1016/s2666-5247(20)30054-9]; no such coordination has been achieved in the USA.

Author: @alavendelm says:

> this might be seen as controversial and you are welcome to remove it if you wish.

I (@rando2) have no issue with it!

Many current efforts to model R~t~ have also led to tools that assist the visualization of estimates in real time or over recent intervals [@url:https://epiforecasts.io/covid; @url:https://rt.live].
These are valuable resources, yet it is also important to note that the estimates arise from models containing many assumptions and are dependent on the quality of the data they use, which varies widely by region.

### Molecular Signatures and Transmission

Genetic variation in SARS-CoV-2 has been used to elucidate patterns over time and space.
Mutations observed in individual SARS-CoV-2 genome sequences can be used to trace transmission patterns and have provided insights during outbreak investigations [@doi:10.1186/s12967-020-02344-6; @doi:10.1093/nsr/nwaa036; @doi:10.1016/j.meegid.2020.104351].
Similar mutations observed in several patients may indicate that the patients belong to the same transmission group.
The tracking of SARS-CoV-2 mutations is recognized as an essential tool for controlling future outbreaks and tracing the path of the spread of SARS-CoV-2.

> Efforts vary widely by country: the UK has coordinated has coordinated a national database of viral genomes [@doi:10.1016/s2666-5247(20)30054-9]; no such coordination has been achieved in the USA.

Reviewer: ```suggestion
Efforts vary widely by country: the UK has coordinated has coordinated a national database of viral genomes [@doi:10/d5mg]; no such coordination has been achieved in the USA.
```
Short DOI is needed.

> 

Author: ```suggestion
```

> Additionally, omics technologies have made it feasible to characterize the host response to the virus.

Author: ```suggestion
Additionally, the rapid development of sequencing technologies over the past decade has made it possible to rapidly characterize the host response to the virus.
```

> Although at present, much of the omics research has utilized _in vitro_ models, such systems-level approaches represent a promising opportunity to characterize the host response to the SARS-CoV-2 virus.

Author: ```suggestion
```

> For example, analysis of protein-protein interaction networks using publicly available data identified some similarities between COVID-19 and tuberculosis infection [@doi:10.1101/2020.03.22.002386].

Author: ```suggestion
For example, proteomics analysis of patient-derived cells revealed candidate genes whose regulation is altered by SARS-CoV-2 infection, suggesting possible approaches for pharmaceutical invention and providing insight into which systems are likely to be disrupted in COVID-19 [@10.1101/2020.03.31.019216].
```

> This finding suggests that insights into COVID-19 may be gained by systems-level analysis of a wide array of infections, even those that are not causes by viruses, as is the case with the bacterium  *M. tuberculosis*.

Author: ```suggestion
```

> As more data is collected, large-scale, omics-based analyses will become more feasible.

Author: @alavendelm says:

> seems out of place and I do not think this paragraph adds any essential points to the discussion?

> As more data is collected, large-scale, omics-based analyses will become more feasible.

Author: It won't let me modify this line, for some reason, but I propose:

> As more patient data becomes available, the biotechnological advances of the 2000s are expected to allow for more rapid identification of potential drug targets than was feasible during the SARS, or even MERS, pandemic.

For example, the acquisition of a furin cleavage site the S1/S2 boundary within the SARS-CoV-2 S protein may be associated with increased virulence.
Additionally, concerns have been raised about the accumulation of mutations within the SARS-CoV-2 species itself, and whether these could influence virulence. <!-- TODO: [cite evolution] -->
The coming of age of genomic technologies has made these types of analyses feasible, and genomics research characterizing changes in SARS-CoV-2 along with temporal and spatial movement is likely to provide additional insights into whether within-species evolution influences the effect of the virus on the human host.
Additionally, the rapid development of sequencing technologies over the past decade has made it possible to rapidly characterize the host response to the virus.

> For example, proteomics analysis of patient-derived cells revealed candidate genes whose regulation is altered by SARS-CoV-2 infection, suggesting possible approaches for pharmaceutical invention and providing insight into which systems are likely to be disrupted in COVID-19 [@10.1101/2020.03.31.019216].

Reviewer: ```suggestion
For example, proteomics analysis of patient-derived cells revealed candidate genes whose regulation is altered by SARS-CoV-2 infection, suggesting possible approaches for pharmaceutical invention and providing insight into which systems are likely to be disrupted in COVID-19 [@doi:10.1101/2020.03.31.019216].
```
